apps/src/lib/kits/maker/boards/microBit/ExternalLed.js

```diff
@@ -1,6 +1,5 @@
 export default class ExternalLed {
   constructor(opts) {
-    //TODO - limit to the GPIO pins on MB?
```

Comment:
We check for valid pins [here](https://github.com/code-dot-org/code-dot-org/blob/staging/apps/src/lib/util/javascriptMode.js#L67).
For micro:bit, valid pins are the GPIO pins (0, 1, 2).